### ISSUE TITLE
Discuss results of 2020 Contributor Survey

### ISSUE THREAD

**Issue by mark-i-m** [rust-lang/compiler-team#318]
# Meeting proposal info

- **Title:** Discuss results of 2020 Contributor Survey
- **Type:** non-technical (mostly)
  
# Summary

Discuss the results of the 2020 Contributor Survey.

There were 409(!) responses. 164 from people who never had a PR merged; 169 from people who have contributed less than once per month; 76 from those who contributed at least once per month. The results have been analyzed by breaking them down into these 3 categories of contributors.

This document has the meat: https://hackmd.io/b-FKKh9DSnah-lj1NVKaAw

# About this issue

This issue corresponds to a meeting proposal for the compiler team
[steering meeting][]. It corresponds to a possible topic of
discussion. You can read more about the [steering meeting procedure
here][proc].

# Comment policy

These issues are meant to be used as an "announcements channel"
regarding the proposal, and not as a place to discuss the technical
details. Feel free to subscribe to updates. We'll post comments when
reviewing the proposal in meetings or making a scheduling decision.
In the meantime, if you have questions or ideas, ping the proposers
on [Zulip] (or elsewhere).

[Zulip]: https://github.com/rust-lang/compiler-team/blob/master/content/about/chat-platform.md
[steering meeting]: https://github.com/rust-lang/compiler-team/blob/master/content/about/steering-meeting.md
[proc]: https://forge.rust-lang.org/compiler/steering-meeting/submit.html
[this template]: https://github.com/rust-lang/compiler-team/blob/master/templates/steering-meeting-proposal.md
[hackmd]: https://hackmd.io/
[gist]: https://gist.github.com/

**Comment by pnkfelix** [comment]
discussed in [today's planning meeting](https://zulip-archive.rust-lang.org/238009tcompilermeetings/09644planningmeeting20200731.html#205596649); scheduled for August 14th meeting slot.

<a target="_blank" href="https://calendar.google.com/event?action=TEMPLATE&amp;tmeid=Mzh2ZWlmNjlnbnBtM212OTI0dXN2aWNodG8gNnU1cnJ0Y2U2bHJ0djA3cGZpM2RhbWdqdXNAZw&amp;tmsrc=6u5rrtce6lrtv07pfi3damgjus%40group.calendar.google.com"><img border="0" src="https://www.google.com/calendar/images/ext/gc_button1_en.gif"></a>

**Comment by pnkfelix** [comment]
After learning that @mark-i-m may be unavailable on the scheduled August 14th, we decided to drop this meeting from the schedule; we'll try to get it into the next steering meeting cycle.

**Comment by nikomatsakis** [comment]
In today's planning meeting, we decided to schedule this for **September 4th**. [Calendar event](https://calendar.google.com/event?action=TEMPLATE&tmeid=NTMwYmxwaDRxZG8xdHJkdm1mdW1jMWxwMmYgNnU1cnJ0Y2U2bHJ0djA3cGZpM2RhbWdqdXNAZw&tmsrc=6u5rrtce6lrtv07pfi3damgjus%40group.calendar.google.com)

**Comment by jyn514** [comment]
This was discussed in https://rust-lang.zulipchat.com/#narrow/stream/238009-t-compiler.2Fmeetings/topic/planning.20meeting.202020.2E09.2E04. Should this issue be closed?

**Comment by nikomatsakis** [comment]
@jyn514 yeah, it should be closed, but usually we close when we create some minutes. I think I said I would do it but I have not :(

**Comment by jyn514** [comment]
I can write up some minutes.

**Comment by jyn514** [comment]
https://hackmd.io/@jynelson/BkqsUyLdP

**Comment by jyn514** [comment]
fixed link: https://hackmd.io/daPDR2Q7Ryey0Q2gqgSM2w
